.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - 'ci/**'
+    - 'master'
+    - 'develop'
```

Comment:
Why should CI only run on these branches?

Reply:
I'm not sure how I landed on this decision. For some reason I was thinking that running ci should always be on purpose, but that doesn't make any sense. Running on every branch seems best :+1: